client/deployment/nginx.conf

```diff
@@ -0,0 +1,21 @@
+events {
+    worker_connections 2000;
+}
+
+http {
+  server {
+    listen 9000;
+
+    location /graphql {
+      # This **needs** to use an environment variable for prod
+      proxy_pass http://server:3000;
+    }
+
+    location / {
+      index /index.html
```

Comment:
should we be serving the assets up through the nginx too?

Reply:
It's usually a pain in other applications as you have to share a volume between the two containers and reset its state appropriately upon a new deploy to avoid older assets being served. Here there is a custom image so it may be easy to copy them in at the right time.

Reply:
I might make configuring nginx properly another ticket, I feel that it's out of scope for this unit of work